projectile.el

```diff
@@ -1026,9 +1026,9 @@ Only buffers not visible in windows are returned."
 
 (defun projectile-paths-to-ignore ()
   "Return a list of ignored project paths."
-  (--mapcat (and (string-prefix-p "/" it)
-                 (directory-file-name it))
-            (cdr (projectile-parse-dirconfig-file))))
+  (-non-nil (--map (and (string-prefix-p "/" it)
+                     (directory-file-name it))
```

Comment:
hmm.. That's weird. I have no customizations on elisp or lisp indentation, although I did recently update prelude. 

Do you know offhand where these settings are? I can't seem to find any customizations for elisp indentation.